.travis.yml

```diff
@@ -1,19 +1,16 @@
 ---
 language: node_js
 node_js:
-  # we recommend testing addons with the same minimum supported node version as Ember CLI
-  # so that your addon works for all apps
-  - "4"
+  - "6"
 
-sudo: false
+sudo: true
```

Comment:
Help me understand this one.  It seems like a critical change to make.

Reply:
The Chrome headless browser started failing in the travis environment. The `sudo` option is for the travis build to correctly use Chrome.

You can see a relevant Github issue here - https://github.com/travis-ci/travis-ci/issues/8836#issuecomment-348227535